doc-content/enterprise-only/integration/sso-client-adapter-proc.adoc

```diff
@@ -71,6 +78,7 @@ In this example:
 * `resource` is the  name for the client that you created.
 * `credential name` is the  secret key for the client you created. You can find the key in the *Credentials* tab on the *Clients* page of the RH-SSO Admin Console.
 * `principal-attribute` is the login name of the user. If you do not provide this value, your User Id is displayed in the application instead of your user name.
+* `kie` is the client resource name. If you want to use client roles with a different client resource name, replace `kie` with the client resource name, for example, `<resource>customClient</resource>`.
```

Comment:
Shouldn't be this part inside the `resource`? `kie` is not name of the configuration tag but it is possible value of the tag `<resource>`.
```
* `resource` is the name for the client that you created. Optional: If you want to use client roles with a different client resource name, replace `kie` with the client resource name, for example, `<resource>customClient</resource>`.
```

Reply:
@mramendi @tomas I believe we dont really need this line here"If you want to use client roles with a different client resource name, replace `kie` with the client resource name, for example, `<resource>customClient</resource>`." Since its quite obvious here that client roles are always taken from the client that is requesting keycloak connection this field is rarely changed for client roles purposes . wdyt ?

Reply:
I don't think I can agree. We can't rely on anything being obvious. I did already join the lines as Tomas requested (see current PR) and I do think this explicit explanation should stay, because not everyone has a perfect understanding of RH-SSO.

Reply:
You are right. But by obvious i meant is , Users always have to use the resource name which they have used while configuring the client in RHSSO. They cannot use any client name. A better way to put this would be "If you want to use client roles, use the same client resource name which you have used while configuring client in RHSSO"

Reply:
@mramendi @RishiRajAnand I am ok with both.